product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/newrelic.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "New Relic"
+---
+
+The New Relic integration is based on the OpenTelemetry Collector, shipping the metrics over the standard OLTP protocol to the New Relic cloud endpoint.
+
+## Pre-requisites
+
+The New Relic account is required to enable this integration.
+
+## Enable New Relic
+
+To enable the **New Relic** integration: 
+1. Select an existing project from the **Projects** page.
+2. Go to **Settings** on the left-side navigation.
+3. Select **Integrations** from the **Settings** drop down list.
+4. Select **New Relic**, a window pops up to provide the details: 
+   - *New Relic API key**
+   - **New Relic Account ID**
+   - **New Relic API Key Name**
+   Provide all the details and select **Save**.
+
+## Metrics
+
+## Costs
```

Comment:
It's _crucial_ that this section has similar wording about responsibility for costs as what is in the DD section.

See also https://enterprisedb.atlassian.net/browse/UPM-29536?focusedCommentId=547574 and https://enterprisedb.atlassian.net/browse/UPM-27110

The docs need to be clear and explicit that any usage-based billing that arises from the data we send to the customer's NR account is _solely_ the customer's responsibility. Even if it's due to an error or oversight on our part.

New Relic bills usage by bytes ingested. Customers _must_ monitor, and configure alerts on, their New Relic data ingestion to reduce the risk of unexpectedly large usage bills.

The NR platform has features to limit usage and ingestion. Customers should review their limits before enabling the BigAnimal integration. See "Administration" -> "Data management" -> "Limits" in the NR UI, and the NR docs:

* [Understand New Relic data limits](https://docs.newrelic.com/docs/data-apis/manage-data/view-system-limits/)
* [Get more detail about your data limits](https://docs.newrelic.com/docs/data-apis/manage-data/query-limits/)

Most or all usage for the BigAnimal integration will be for metrics data ingest, though logs ingestion may be added at a future time so logs limits should also be checked.